lib/rubocop/rspec/config_formatter.rb

```diff
@@ -15,7 +15,9 @@ def initialize(config, descriptions)
       end
 
       def dump
-        YAML.dump(unified_config).gsub(EXTENSION_ROOT_DEPARTMENT, "\n\\1")
+        YAML.dump(unified_config)
+          .gsub(EXTENSION_ROOT_DEPARTMENT, "\n\\1")
+          .gsub(/^(\s+)- /, '\1  - ')
```

Comment:
Add two extra spaces of indentation to every line starting with `- ` or `<some spaces>- `.

Reply:
Aha, this is where the flat list indentation was coming from! 💯